src/main/flight/mixer.c

```diff
@@ -728,6 +730,10 @@ float applyThrottleLimit(float throttle)
                 return MIN(throttle, throttleLimitFactor);
         }
     }
+    
+    if (gyroYawSpinDetected()) {
+        throttle = 0.0f;
+    }
```

Comment:
I don't think this code should be in the `applyThrottleLimit()` function.  Probably should be directly in `mixTable()` right after the section that calls `applyThrottleLimit()`.  Also this code should be wrapped in the `USE_...` defines.

Reply:
To be honest I wasn't sure of the best place.  Those kind of decisions require more expertise than I have.  I'd be grateful if you could move it to the optimal location, and put it in the right defines.